cmd/cp-main.go

```diff
@@ -73,9 +73,25 @@ var (
 			Name:  "disable-multipart",
 			Usage: "disable multipart upload feature",
 		},
+		cli.StringFlag{
+			Name:  rmFlag,
+			Usage: "retention mode to be applied on the object",
+		},
+		cli.StringFlag{
+			Name:  rdFlag,
+			Usage: "retention duration for the object",
+		},
+		cli.StringFlag{
+			Name:  lhFlag,
+			Usage: "apply legal hold to the copied object",
```

Comment:
Could you add possible values here ? (I feel like it doesn't have any value here)